crypto/src/keys/fvk.rs

```diff
@@ -81,6 +86,15 @@ impl FullViewingKey {
     pub fn spend_verification_key(&self) -> &VerificationKey<SpendAuth> {
         &self.ak
     }
+
+    /// Hashes the full viewing key into a [`FullViewingKeyHash`].
+    pub fn hash(&self) -> FullViewingKeyHash {
+        let hash_result = prf::expand(b"Penumbra_HashFVK", &self.nk.0.to_bytes(), self.ak.as_ref());
```

Comment:
This is the same way that the outgoing viewing key and diversifier key are derived from the FVK, but with a different label to ensure different outputs.